_RFCs/0009-plan-configuration-complexity.md

```diff
@@ -0,0 +1,134 @@
+- Change Name: plan_configuration_complexity
+- Start Date: 2018-08-29
+- RFC PR: 
+- Core-Plans Issue: 
+
+# Summary
+[summary]: #summary
+
+When packaging software with Habitat, we need to ensure we have an agreed level of complexity with regards to the configuration that is shipped. This RFC proposes a standard by which plans can be developed, and kept simple and functional.
+
+# Motivation
+[motivation]: #motivation
+
+As the [core-plans][core-plans] repository continues to evolve and grow, more users are coming with new and useful use cases for the software that is packaged with Habitat. In order to ensure we maintain the core plans as simple, functional, testable and model examples of software pacakging, we must ensure complexity is kept to a minimum. This will encourage simpler configuration in [core-plans][core-plans] plans, and encourage users to use the configuration plan pattern to extend configuration where necessary.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+A habitat [core-plan] is a plan that is supported and published by core-contributors of the Habitat community. These are often relied upon by other plans, and as part of the Habitat build process itself.
+
+A core-plan's primary responsibility is to ensure the binaries and associated files responsible for running the commands or services are correctly built, packaged and released. In addition, for service pacakges, a minimal configuration representing the minimal/default service configuration is shipped to ensure that services can start in a simple state. This simple/minimal state is representative of the default configuration and minimal configuration required to get the service running.
```

Comment:
Slight typo - "pacakges" should be "packages"

Reply:
Done!